Makefile

```diff
@@ -26,6 +26,10 @@ hooks:
 	@chmod 755 ./.git/hooks/pre-push
 	@echo "Hooks installed successfully!"
 
+db_create:
+	@GO_ENV=development go run ${DB_SCRIPT_ENTRYPOINT} create
+db_drop:
+	@GO_ENV=development go run ${DB_SCRIPT_ENTRYPOINT} drop
```

Comment:
We should add the targets to `DB_TARGETS` at the top of this file as well. That way, they'll be `.PHONY`-ed, and we can always set configurations that apply to all DB targets if we want to.